examples/main.rs

```diff
@@ -19,9 +20,14 @@ type EdgeData = Bytes;
 pub struct Node {
     id: Id,
     data: NodeData,
+    node_type: types::NodeType,
```

Comment:
Why is this separate from `NodeData`?

Reply:
Ah, simply for conveniency -- this example is using blob of bytes as our `NodeData`, but really what I wanted to use here was the concrete node data, but I also didn't want to throw away the part of the code you did write. Suggestions? I am totally fine with having `NodeData` be `types::NodeData`, and then remove this `node_type` field.

Reply:
Yeah I'd just say add `type NodeData = NodeType` for eg.